osp/core/session/db/db_wrapper_session.py

```diff
@@ -161,3 +167,29 @@ def _is_cuds_iri_ontology(iri):
                                rdflib.OWL.Class}):
                 return False
         return True
+
+    def _unreachable_warning(self, root_obj: Union[rdflib.URIRef, uuid.UUID]):
+        """Raises a warning when there are unreachable cuds.
+
+        Gets a list of all the CUDS objects in the session which are
+        unreachable from the specified `root_obj`. If there are any, then
+        raises a warning that lists some of the unreachable CUDS objects.
+
+        Args:
+            root_obj (Union[URIRef, UUID]): The root object with respect to
+                which objects are deemed reachable or unreachable.
+        """
+        unreachable = self._registry._get_not_reachable(root_obj, rel=None)
+        max_cuds_on_warning = 5
+        if len(unreachable) > 0:
+            warning = "Some CUDS objects are unreachable " \
```

Comment:
In the message, we cal already suggest a solution (i.e., add the object to the wrapper object)

Reply:
Done.